css_stylesheet.css

```diff
@@ -0,0 +1,70 @@
+body {
+	font-family: "Arial", sans-serif;
+	margin: 5px;
+	padding: 5px;
+	margin-left: auto;
+	margin-right: auto;
+	width: 1200px;
+}
+
+
+header {
+	background-color: #64E6BE;
+	overflow: auto;
+	border: solid 5px #343A40;
+	border-radius: 5px;
+}
+
+.buttonsSentLeft {
+	float: left;
+}
+
+.contactAnchors {
+	list-style-type: none;
+ }
+
+.email {
+	font-weight: 900;
+	text-decoration: none;
+	font-family: "Courier New", serif;
+	color: #343A40;
+}
+
+.email:hover {
+	text-decoration-line: underline;
+	/*text-shadow: 1px 1px;*/
+	font-family: "Courier New", serif;
+	color: #343A40;
+}
+
+.number {
+	font-weight: 900;
+	text-decoration: none;
+	font-family: "Courier New", serif;
+	color: #343A40;
+}
```

Comment:
This looks like it contains the same css as the .email class, could they be combined into a single class? Keep the code DRY.